.changeset/loud-books-teach.md

```diff
@@ -0,0 +1,22 @@
+---
+'xstate': major
+---
+
+Target resolution improvements: delimiters in state node IDs now resolve to the state node from the path relative to the resolved state node ID:
```

Comment:
this isn't new as far as I can tell: 
https://codesandbox.io/s/dawn-brook-tbt0ht?file=/src/index.js

Reply:
👋

Reply:
Removed